ecs-init/cache/cache_test.go

```diff
@@ -124,11 +125,14 @@ func TestDownloadAgentDownloadMD5Failure(t *testing.T) {
 	mockGetter := NewMockhttpGetter(mockCtrl)
 
 	mockFS.EXPECT().MkdirAll(config.CacheDirectory(), os.ModeDir|0700)
-	mockGetter.EXPECT().Get(config.AgentRemoteTarballMD5()).Return(nil, errors.New("test error"))
+	mockGetter.EXPECT().Get(config.AgentRemoteTarballMD5(config.DefaultRegionName)).Return(nil, errors.New("test error"))
```

Comment:
Can we be more strict and the check the ordering here (Using [`gomock.InOrder`](https://godoc.org/github.com/golang/mock/gomock#InOrder))?